### PR TITLE
Error screen

### PR DESCRIPTION
**Pull request by Damidjo**
An error screen appears when the network fails to connect.

### CODE REVIEW

File: app/src/main/res/layout/error_screen.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    android:orientation="horizontal"
+    tools:context=".ErrorScreen">
+
+
+    <ImageView
+        android:id="@+id/error_image"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_centerHorizontal="true"
+        android:layout_marginTop="150dp"
```
Comment (ShAdOwNeT88) [inline app/src/main/res/layout/error_screen.xml]: I don't think that we need to have all those dimensions hardcoded, please export them into dimens file

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,11 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="app_name">METEO</string>
+    <string name="error_message">      C’è stato un problema con
+        il caricamento dei tuoi dati.
+        Verifica la tua rete e riprova.
```
Comment (ShAdOwNeT88) [inline app/src/main/res/values/strings.xml]: It's unnecessary to paste strings with new line like this, they will be not rendered into the ui.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
+    </string>
     <string name="palermo_sic">Palermo, Sicilia</string>
     <string name="prossimi_5">PROSSIMI 5 GIORNI</string>
     <string name="hello_blank_fragment">Hello blank fragment</string>
     <string name="home">Home</string>
     <string name="Today">Today</string>
     <string name="Tomorrow">Tomorrow</string>
     <string name="Search">Search</string>
+    <string name="riprova">Riprova</string>
```
Comment (ShAdOwNeT88) [inline app/src/main/res/values/strings.xml]: Why we are mixing Eng and It?

**Review by ShAdOwNeT88 — CHANGES_REQUESTED** [review]

**Comment by Damidjo** [comment]
Added dimen.xml with dimension of error-screen.
Solved language issues
Solved string issues

File: app/src/main/res/values/dimen.xml [hunk 4]
```diff
@@ -4,4 +4,16 @@
...
+    <dimen name="padding_start">60dp</dimen>
+    <dimen name="padding_end">20dp</dimen>
+
+    <!-- TextView for error_message -->
+    <dimen name="marginTop">20dp</dimen>
+    <dimen name="textSize">17sp</dimen>
+
+    <!-- TextView for retry_button -->
+    <dimen name="textMarginTop">20dp</dimen>
+</resources>
```
Comment (ShAdOwNeT88) [inline app/src/main/res/values/dimen.xml:20]: If we have three dimensions at 20dp, why do we merge them into one single dimension named `margin_20`

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -1,11 +1,15 @@
...
-    <string name="Tomorrow">Tomorrow</string>
-    <string name="Search">Search</string>
+
+
+    <string name="Home">Home</string>
+    <string name="Oggi">Oggi</string>
+    <string name="Domani">Domani</string>
+    <string name="Cerca">Cerca</string>
+    <string name="Riprova">Riprova</string>
 </resources>
```
Comment (ShAdOwNeT88) [inline app/src/main/res/values/strings.xml]: Maybe we need to have the en strings too?

**Review by ShAdOwNeT88 — CHANGES_REQUESTED** [review]

**Pull request merged**